packages/ui/src/working-groups/components/Roles/RolesList.tsx

```diff
@@ -80,13 +97,17 @@ const RolesListItem = ({ worker }: { worker: Worker }) => {
         </OpenItemSummaryColumn>
         <OpenItemSummaryColumn>
           <TextInlineBig>
-            <TokenValue value={worker.rewardPerBlock} />
+            <TokenValue value={earnings} />
           </TextInlineBig>
           <ToggleableSubscriptionWide>Earned total</ToggleableSubscriptionWide>
         </OpenItemSummaryColumn>
         <OpenItemSummaryColumn>
           <TextInlineBig>
-            <TokenValue value={earnings} />
+            <Tooltip
+              tooltipText={<NextPaymentValue>{payout?.gte(BN_ZERO) ? payout?.toString() : '–'}</NextPaymentValue>}
+            >
+              <NextPaymentValue>{payout?.gte(BN_ZERO) ? payout?.toString() : '–'}</NextPaymentValue>
+            </Tooltip>
```

Comment:
This not very DRY. No need for the tooltip if it shows exactly the same:
```suggestion
              <NextPaymentValue>{payout?.gte(BN_ZERO) ? payout?.toString() : '–'}</NextPaymentValue>
```

Reply:
Nice catch I created an issue for this: #4323